### PR TITLE
Update 1010-CG and 10-10EZ Sidekiq Documentation

### PR DESCRIPTION
**Pull request by coope93**
Update sidekiq info due to a recent change. Also formatted the docs a bit.

### CODE REVIEW

File: products/caregivers/10-10CG Form/Existing Monitors and Alerts.md [hunk 1]
```diff
@@ -1,51 +1,83 @@
...
 - No Action needed, this alert is letting us know that volumes are back to normal
+
 ## Monitor: 1010CG submission job has failed retries
+
 ### Triggered: 1010CG submission job has failed ten retries
+
 #### Alert Reason
-- At least one application submission has been retried 10 times, and has failed.  The submission will retry 4 more times before failing completely.
+
+- At least one application submission has been retried 10 times, and has failed. The submission will retry 6 more times before failing completely.
```
Comment (coope93) [inline products/caregivers/10-10CG Form/Existing Monitors and Alerts.md:38]: This was changed from 14 to 16 for most sidekiq jobs to allow for handling COLA outages. https://github.com/department-of-veterans-affairs/vets-api/pull/19289 for the PR.

File: products/caregivers/10-10CG Form/Existing Monitors and Alerts.md [hunk 2]
```diff
@@ -1,51 +1,83 @@
...
-- Determine if a maintenance banner needs to be in place to stop further failures, 
+- Determine if a maintenance banner needs to be in place to stop further failures,
 - Communicate on the **#vecms-carma_vadotgov_development_external** Slack channel
+
 ### Triggered: 1010CG submission job has failed with no retries left
+
 #### Alert Reason
-- At least one application submission has been retried 14 times over the course of ~1 day, has failed and will not be retried again.
+
+- At least one application submission has been retried 16 times over the course of ~2d 1h 47m 12s, has failed and will not be retried again.
```
Comment (coope93) [inline products/caregivers/10-10CG Form/Existing Monitors and Alerts.md:51]: https://github.com/department-of-veterans-affairs/vets-api/pull/19289

File: products/health-care/application/va-application/10-10EZ Form/Existing Monitors and Alerts.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ All alerts are set up to populate the **#health-tools-1010-apm** slack channel.
 ## Monitor: 1010EZ submission job has failed retries
 ### Triggered: 1010EZ submission job has failed with no retries left
 #### Alert Reason
-- At least one application submission has been retried 24 times, has failed and will not be retried again.
+- At least one application submission has been retried 16 times, has failed and will not be retried again.
```
Comment (coope93) [inline products/health-care/application/va-application/10-10EZ Form/Existing Monitors and Alerts.md:32]: https://github.com/department-of-veterans-affairs/vets-api/pull/19289

**Review by Mitch-A6 — APPROVED** [review]
LGTM.

**Comment by hdjustice** [comment]
@coope93 - What was the reason for 16 retries?  I thought we went with 14 (just over 24 hrs).

**Comment by coope93** [comment]
@hdjustice [slack thread for the reasons](https://dsva.slack.com/archives/C053U7BUT27/p1730909263999689?thread_ts=1730731734.405619&cid=C053U7BUT27) and [PR where it was changed](https://github.com/department-of-veterans-affairs/vets-api/pull/19289). 

TLDR: They wanted to ensure that due to the COLA updates and downtime that any sidekiq jobs would retry _past_ the downtime amount. So all/most sidekiq jobs were updated to 16 retires to allow for extra time for them to retry beyond the outage time.  

All 3 of our main submission jobs (cg/ez/ezr) were updated to 16 retries.

**Comment by hdjustice** [comment]
got it, thank you for this context!
